docs/advanced-node-scheduling.asciidoc

```diff
@@ -253,22 +248,13 @@ spec:
                   operator: In
                   values:
                   - europe-west3-b
-  updateStrategy:
-    groups:
-    - selector:
-        matchLabels:
-          nodesGroup: group-a
-    - selector:
-        matchLabels:
-          nodesGroup: group-b
 ----
 
 This example relies on:
 
 - nodes from each zone being labeled accordingly. `failure-domain.beta.kubernetes.io/zone` link:https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#interlude-built-in-node-labels[is standard], but any label can be used.
 - node affinity for each group of nodes set to match the Kubernetes nodes zone.
 - Elasticsearch configured to link:https://www.elastic.co/guide/en/elasticsearch/reference/current/allocation-awareness.html#allocation-awareness[allocate shards based on node attributes]. Here we specified `node.attr.zone`, but any attribute name can be used. `node.attr.rack_id` is another common example.
```

Comment:
This is slightly tangential to this PR but it seems like "node" is used interchangeably in this section without qualification. I think it might be better to identify which type of node is being talked about to avoid confusion.

Reply:
Good point! I created https://github.com/elastic/cloud-on-k8s/issues/1851 to track that.